docs/developers/default-request.md

```diff
@@ -0,0 +1,44 @@
+# Server Based Global Default Request
+
+This allows a defaut stored request to be defined that allows the server to set up some defaults for all incoming requests. A request specified stored request will override these defaults, and of course any options specified directly in the stored request override both. The default stored request is only read on server startup, it is meant as an installation static default rather than a dynamic tuning option.
+
+A common use case is to "hard code" aliases into the server. This saves having to specify them on all incoming requests, and/or on all stored requests. To help support automation and alias discovery we can flag that any aliases found in the file be added to the bidder info endpoints.
+
+## Config Options
+
+Three config options are exposed to support this feature.
+```
+default_request:
+    type: "file"
+    filesystem:
+        filename : /path/to/aliases.json
+    alias_info     : false
+```
+
+The `filename` option is the path/filename of a JSON file containing the default stored request JSON as documented in the [openrtb2 docs](../endpoints/openrtb2/auction.md) and [stored request docs](stored-request.md)
+```
+{
+    "tmax": "<auction_timeouts_ms.default>",
+    "regs": {
+        "ext": {
+            "gdpr": <gdpr.usersync_if_ambiguous>
+        }
+    },
+    "etc": {
+        "prebid": {
+            "aliases": {
+                "districtm": "appnexus"
+            }
+        }
+    }
+}
+```
+This will be JSON merged into the `ext` element of the incomming requests to define common aliases for the server.
+
+The `info` option determines if the alised bidders will be exposed on the `/info` endpoints. If true the alias name will be added to the list returned by
+`/info/bidders` and the info JSON for the core bidder will be coppied into `/info/bidder/{biddername}` with the addition of the field 
```

Comment:
Any thoughts on `/bidders/params` here?

If the goal is to make aliases seem just like real bidders when this flag is flipped, it seems to me like they should appear there too... but I'm not totally sold on it.

Reply:
Yes, missed that, since it wasn't handled in `bidders.go`